Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:12-slim
```

Comment:
This `Dockerfile` was heavily influenced by the existing WIP changes in https://github.com/vercel/cosmosdb-server/compare/master...ridhoq:docker.